package.json

```diff
@@ -28,7 +28,8 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "format": "prettier --write \"./**/*.{js,css,json,scss}\"",
-    "check-format": "prettier -c \"./**/*.{js,css,json,scss}\""
+    "check-format": "prettier -c \"./**/*.{js,css,json,scss}\"",
+    "prepare": "husky install"
```

Comment:
No need in the main project now

Reply:
let me remove it

Reply:
check format removed however husky is based on git which is initial at the root level we can take it next project at final step